src/main/java/net/minecraftforge/eventbus/api/EventListenerHelper.java

```diff
@@ -32,10 +32,10 @@
 
 public class EventListenerHelper
 {
-    private static final LockHelper<Class<?>, ListenerList> listeners = new LockHelper<>(new IdentityHashMap<>());
+    private static final LockHelper<Class<?>, ListenerList> listeners = new LockHelper<>(IdentityHashMap::new);
     private static final ListenerList EVENTS_LIST = new ListenerList();
-    private static final LockHelper<Class<?>, Boolean> cancelable = new LockHelper<>(new IdentityHashMap<>());
-    private static final LockHelper<Class<?>, Boolean> hasResult = new LockHelper<>(new IdentityHashMap<>());
+    private static final LockHelper<Class<?>, Boolean> cancelable = new LockHelper<>(IdentityHashMap::new);
+    private static final LockHelper<Class<?>, Boolean> hasResult = new LockHelper<>(IdentityHashMap::new);
```

Comment:
Could rename all of these to upper snake case while you're at it.

Reply:
I'd like to keep the style cleanups after the rest of the reworks if you don't mind? There's a lot more stuff to fix with the style :D